src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -37,6 +37,11 @@ import VariantCountCache from "./VariantCountCache";
 
 type PageMode = 'patient' | 'sample';
 
+class OncokbDefault implements IOncoKbData {
+    sampleToTumorMap = {};

```

Comment:
no, this is a shorthand that typescript supports.  you can define and initialize instance props in once place.